articles/quickstart/spa/react-beta/_includes/_centralized_login.md

```diff
@@ -0,0 +1,404 @@
+<!-- markdownlint-disable MD002 MD041 MD034 -->
+
+<%= include('../../_includes/_login_preamble', { library: 'React' }) %>
+
+## Create a Sample Application
+
+::: note
+The following tutorial creates a new React application using `create-react-app`, and presents some common ways to build React applications, in terms of its structure and naming conventions. If you are using this guide to integrate the Auth0 SDK into your own React application, you may need to adjust some of the steps to suit your scenario.
+:::
+
+If you don't already have an application to hand, you can create one using the [Create React App](https://facebook.github.io/create-react-app/) CLI tool. Using the terminal, find a location on your drive where you want to create the project and run the following commands:
+
+```bash
+# Create the application using create-react-app
+npx create-react-app my-app
+
+# Move into the project directory
+cd my-app
+```
+
+(npx comes with npm 5.2+ and higher, see [instructions for older npm versions](https://gist.github.com/gaearon/4064d3c23a77c74a3614c498a8bb1c5f))
+
+### Install initial dependencies
+
+After creating a new React app using `create-react-app` install `react-router`, which doesn't come as standard with the boilerplate project. The [Auth0 Client SDK](https://github.com/auth0/auth0-spa-js) should also be added.
+
+Install these two packages using the following command in the terminal:
+
+```bash
+npm install react-router-dom @auth0/auth0-spa-js
```

Comment:
Should we use the "--save-dev" param?

Reply:
We do this for the sample app. For the sample being described in the documentation, it doesn't really make any difference.